six/modules/c++/six.sicd/tests/test_add_additional_des.cpp

```diff
@@ -0,0 +1,252 @@
+/* =========================================================================
+* This file is part of six.sidd-c++
+* =========================================================================
+*
+* (C) Copyright 2004 - 2016, MDA Information Systems LLC
+*
+* six-c++ is free software; you can redistribute it and/or modify
+* it under the terms of the GNU Lesser General Public License as published by
+* the Free Software Foundation; either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU Lesser General Public License for more details.
+*
+* You should have received a copy of the GNU Lesser General Public
+* License along with this program; If not,
+* see <http://www.gnu.org/licenses/>.
+*
+*/
+
+#include <six/NITFWriteControl.h>
+#include <six/Types.h>
+#include <six/XMLControlFactory.h>
+#include <six/sicd/Utilities.h>
+
+
```

Comment:
Never mind, I know how Google works.



This is giving a lot of compiler errors like
From: Adam Sylvester [mailto:notifications@github.com]
Sent: Tuesday, May 24, 2016 8:06 PM
To: ngageoint/six-library <six-library@noreply.github.com>
Cc: Jonathan Means <Jonathan.Means@mdaus.com>; Author <author@noreply.github.com>
Subject: Re: [ngageoint/six-library] Add method to allow adding extra DES to NITFWriteControl (#76)


In six/modules/c++/six.sicd/tests/test_add_additional_des.cpp<http://cp.mcafee.com/d/5fHCMUp40USyMUMyOMe7fCXCSm6hMUQsLLCTNPb3a9EVd79J6XRTD3hPsQsL6QkmkkTPhOUM-r4qMSpfosH-ME2BkdrFOVJl3mWsKrs7mZxws_R-phoLstRXBQQhMXD64TCn-EyCJtdmXPDaxVZicHs3jq9JcTvAn3hOYyyODtUTsSjDdqymohBcCSgVv84t8_mk9VigunfAgzIiSbdDDUes0N2lbdQFaI1mbz31EVsvjhdAaJMJZ00Lcfc8IBSm4tbydorcDIel_okelgRKDbwGx4jUzapdIxOVJZWVEVdwLQzh0qmU4t8_mk9VigunfAgzIiNFtd40wwwq85vCjGwq89Rd46DDRc6y3k49g8Cy3M-std40sd31EwDkQg0Md18K0IpjYiH2sqDCy0Qk0oesd59IQsTdR1SdgFGB2sD>:

> +* but WITHOUT ANY WARRANTY; without even the implied warranty of

> +* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the

> +* GNU Lesser General Public License for more details.

> +*

> +* You should have received a copy of the GNU Lesser General Public

> +* License along with this program; If not,

> +* see <http://www.gnu.org/licenses/>.

> +*

> +*/

> +

> +#include <six/NITFWriteControl.h>

> +#include <six/Types.h>

> +#include <six/XMLControlFactory.h>

> +#include <six/sicd/Utilities.h>

> +

> +

Put all these in an unnamed namespace

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<http://cp.mcafee.com/d/5fHCMUe418qdEIec8II3xPVKVJBxAsed7bXVJYsOMOyqejhOrhKZtVMQsTd7bNJ55B5dYQsKcfCN6IdCjS7a_Ia0Fl3mWsKrlgRKDbCT1RLoo7fZvCkmbT7tuVtd4seVNxdVB_G8FHnjlKYVOEuvkzaT0QSCrjdTV5MQsL8EIFTudTdAVPmEBC4pj9JAenO17ifRB2ukA7BPV48X4JyPpV-3DctAvZc8nd7atTC1MWMVZ15cwPZT3hO-_XDEEfL3DvsKPp-W86hsood7bzWq9IxlK5LE05VxVx5AKOMzFshH3pAZxOLX2xOG6JQVs5k8yv4pj9JAendLLnd79I5-Aq83iT0zF7WOxfai3OVYy4tymdbFEw4443h0HYOtk3h1eFEwQY-FwQgqwxa14Qgu7PzFEw3xEod44WCy061E95M5zavylojzkYQg6yw31PxEFdCzCVIMBtzQMY7NC3>


The information contained in this communication is confidential, is intended only for the use of the recipient(s) named above, and may be legally privileged. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited. If you have received this communication in error, please re-send this communication to the sender and delete the original message or any copy of it from your computer system.